bayesian/utility/security_vendor.py

```diff
@@ -0,0 +1,355 @@
+# Copyright © 2020 Red Hat Inc.
```

Comment:
rename to v2_response_builder.py

Reply:
v2_ca_response_builder, stack analysis can have another module.

Reply:
Ack!